README.md

```diff
@@ -101,11 +101,13 @@ In order to align the `knative.dev` dependencies, knobots will perform PRs like
 the command `./hack/update-deps.sh --upgrade --release 0.20` and committing all
 the content.
 
-If no dependency bump PR is available, you can:
+If no dependency bump PR is available, you can either:
 
 - Manually trigger the generation of these PRs starting the
-  [Knobots Auto Updates workflow](https://github.com/knative-sandbox/knobots/actions?query=workflow%3A%22Auto+Updates%22)
-  and wait for the PR to pop in the repo you need.
+  [Knobots Auto Update Deps](https://github.com/knative-sandbox/knobots/actions/workflows/auto-update-deps.yaml)
+  and wait for the PR to pop in the repo you need. Note you have to change the
+  field `If true, send update PRs even for deps changes that don't change vendor. Use this only for releases.`
+  to true, because in some cases there no code changes in the vendor.
```

Comment:
When I ran these workflows I also set the name of the repository (knative/networking, knative/caching, knative-sandbox/reconciler-test) to avoid generating too many PRs (since the mentioned boolean field may generate the PRs for non-change as well).

Reply:
I think it should be okay to run this for all the repos; it will lock their `go.mod` onto the release as soon as it's available, rather than doing it late-bound like it is today.